steampy/market.py

```diff
@@ -52,6 +52,21 @@ def get_my_market_listings(self) -> dict:
         listings = get_market_listings_from_html(response.text)
         listings = merge_items_with_descriptions_from_listing(listings, listing_id_to_assets_address,
                                                               assets_descriptions)
+        if '<span id="tabContentsMyActiveMarketListings_end">' in response.text:
+            n_showing = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_end">', '</span>'))
+            n_total = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_total">', '</span>'))
+            if n_total > n_showing:
```

Comment:
I think the IF at line 58 is redundant because if there are less or 10 items the IF at line 55 would be False.

And no, the IF at line 55 specifically check if there are those little arrows: http://i.imgur.com/5ZAXIjt.png